### DISCUSSION TITLE
Burp collaborator support removed

### DISCUSSION THREAD

**Discussion by fopina** [projectdiscovery/nuclei#1362] [Q&A] [answered]
Hey, with all the log4j fever this weekend bringing interactsh service down, I fell back to burp collaborator and had to extract the hits manually.

Thinking that it'd be nice to have that support in nuclei as well, found out that it did have but was removed - https://github.com/projectdiscovery/nuclei/commit/f5b9bed40a69eb3c79393d474e0760bbac8ea5b0

Why was that? Some strategic decision towards focusing on interactsh? Or lack of interest in the flag and supporting it?

**Comment by ehsandeep** [comment, marked as answer]
@fopina it was removed as burp  collaborator does not have official integration support and it was also a reason why we started to work on interactsh.

Initially the collaborator support was added with hack where users have to extract the `biid` by intercpeting the collaborator request separately, so maintaining unofficial support + pre-requisite of `biid` outside of the template + making burp collaborator UI client unusable within burp (as same `biid` can not be used in 2 place) was not feasible and deprecated.

**Reply by fopina** [reply]
made perfect sense, thanks!
